src/containers/BookFormRedux.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form';
+
+const renderInput = ({ input, label, type, meta: { touched, error, warning } }) => (
+  <div>
+    <label>{label}</label>
+    <div>
+      <input {...input} placeholder={label} type={type} />
+      {touched && (error && <span style={{ color: 'red' }}>{error}</span>)}
+    </div>
+  </div>
+);
+
+const renderTextarea = ({ input, label, type, meta: { touched, error, warning } }) => (
+  <div>
+    <label>{label}</label>
+    <div>
+      <textarea {...input} placeholder={label} type={type} />
+      {touched && (error && <span style={{ color: 'red' }}>{error}</span>)}
+    </div>
+  </div>
+);
+
+const validate = values => {
+  const errors = {}
+  if (values.isbn === null || values.isbn === undefined || values.isbn === '') {
+    errors.isbn = 'ISBN ungültig';
+  }
+  if (values.title === null || values.title === undefined || values.title === '') {
+    errors.title = 'Title ungültig';
+  }
+  return errors
+}
+
+
+const BookFormRedux = ({ handleSubmit, cancelPath }) => (
+  <form onSubmit={handleSubmit}>
+    <Field name="isbn" label="ISBN" component={renderInput} type="text"/>
```

Comment:
I haven't checked, but I assume since you pass itt as a component it should be an upper cased component name, because if used properly by the lib, it will be displayed in the dev tools, also it seems recommended that way by the docs https://redux-form.com/8.0.4/docs/api/field.md/#usage

Reply:
There is a difference between component as a prop and render prop.

Reply:
👍 Done in [PR](https://github.com/workshops-de/reactjs-workshop/pull/4)